### ISSUE TITLE
Feature Request: Is FocusPRO® 6000 5-1-1 Thermostat supported

### ISSUE THREAD

**Issue by chritd** [d0n4v4nb3ck3r/homebridge-honeywell-home#808]
### Problem

The thermostat does not show up after linking

### Solution

Access this thermostat vi home bridge/homekit

**Comment by donavanbecker** [comment]
Do you have any logs to provide me?

**Comment by chritd** [comment]
Sorry the late reply

I updated to your latest version and reinstated the 5 lyrics I have

In the log I get 1 warning message. If I see it correctly. Other wise no messages from Resideo in the log. The 26.5 matches that I sent the thermostat to 80 to see if its working. 

[1/23/2024, 9:41:30 PM] [Resideo] Room Sensor Thermostat: golden gate set request ({"mode":"Heat","autoChangeoverActive":[Undefined],"heatSetpoint":80,"coolSetpoint":[NaN]}) to Resideo API.

[1/23/2024, 9:41:46 PM] [homebridge-resideo] This plugin generated a warning from the characteristic 'Heating Threshold Temperature': characteristic was supplied illegal value: number 26.5 exceeded maximum of 25. See https://homebridge.io/w/JtMGR for more info.

I then set the temperature to 70f and got following in the log

[1/23/2024, 9:47:17 PM] [Resideo] Room Sensor Thermostat: golden gate set request ({"mode":"Heat","autoChangeoverActive":[Undefined],"heatSetpoint":70,"coolSetpoint":[NaN]}) to Resideo API.

What I am wondering is why the Resido messages are in Fahrenheit and homebridge-resideo in Celsius,

I'll report tomorrow if the system runs crazy again and starts to heat everything up to 80 and above although all thermostats are set to 70 (that was the issue the last times I tried your plugin)
